src/components/asset-list/RecyclerAssetList2/WrappedSavingsRow.tsx

```diff
@@ -13,6 +13,7 @@ export default React.memo(function WrappedSavingsRow({
 
   const props = useMemo(() => {
     const found = savings.find(
+      // @ts-expect-error TODO: add typings
```

Comment:
Is this comment to stay? If yes, how was it passing CI before 👀?

Reply:
:broom::broom:
no, it's just a warning; after Marco's covered all places in the application, those used to be everywhere
i removed it